app/assets/stylesheets/components/_icon.scss

```diff
@@ -12,3 +12,22 @@
     background-image: url(image-path('ico-print.svg'));
   }
 }
+
+.ico-absolute {
+  background-repeat: no-repeat;
+  background-size: $h4;
```

Comment:
I'm not sure actually. I can see the benefit for having sized classes, like `ico-absolute-lg`, but I don't know exactly what api i would want to expose to the devs using the classes. Plus, altering the size of the icon would necessitate changing the top and left offsets (probably with a calculation?).

Does this ultimately make sense as a mixin? You add this to an existing class to give it a pseudo element with an icon?

What do you mean by bound to `h4`?